packages/Liquid-Core.package/LiquidParticipantMenu.class/instance/sendAnswer.st

```diff
@@ -1,3 +1,7 @@
 buttons
 sendAnswer
-	self poll addAnswerSet: answerSet.
\ No newline at end of file
+	(LiquidParticipantMenu answeredPollIds ifAbsentAdd: self poll id)
```

Comment:
[sand] I had to look into `ifAbsentAdd` to find out what it returns, maybe we could assign it to some name to make it easier to read.
```suggestion
	answerIsNew := LiquidParticipantMenu answeredPollIds ifAbsentAdd: self poll id
	answerIsNew ifTrue: ...
```

Reply:
I feel like temp variables are not regarded very well in Squeak, but I agree and have applied your feedback